finetune/adapter_v2.py

```diff
@@ -54,13 +54,22 @@
 
 
 def main(
-    data_dir: Path = Path("data/alpaca"),
-    checkpoint_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b"),
-    out_dir: Path = Path("out/adapter_v2/alpaca"),
-    precision: Literal["bf16-true", "32-true", "bf16-mixed"] = "bf16-true",
-):
+        data_dir: Path = Path("data/alpaca"),
+        checkpoint_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b"),
+        out_dir: Path = Path("out/adapter_v2/alpaca"),
+        precision: Literal["bf16-true", "32-true", "bf16-mixed"] = "bf16-true",
+        # by default use the same block size as the checkpoint model
+        max_seq_length : Optional[int] = None):
```

Comment:
I don't think we need to expose this as an argument. We can always get the block size

Reply:
See above question (are there cases where you want to change the max_seq_length for fine-tuning?)